iosApp/iosApp/ContentView.swift

```diff
@@ -194,7 +206,7 @@ struct ContentView: View {
     }
 
     @ViewBuilder
-    var navSheetContents: some View {
+    var sheetContents: some View {
```

Comment:
sheetContents uses `lastSafe` which defaults to `.nearby`. The easiest change seemed to be only use sheetContents if we know the default tab is loaded, and therefore `lastSafe` will accurately reflect whether nearby or favorites is last.